docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -1,19 +1,19 @@
 Follow these guidelines for setting up freeCodeCamp locally on your system. This is highly recommended if you want to contribute regularly.
 
-For some of the contribution workflows, you need to have freeCodeCamp running locally. For example, previewing coding challenges or debugging and fixing bugs in the codebase.
+Some of these contribution workflows – like fixing bugs in the codebase or curriculum – you'll need to run freeCodeCamp locally on your computer.
```

Comment:
```suggestion
Some of these contribution workflows – like fixing bugs in the codebase or curriculum – need you to run freeCodeCamp locally on your computer.
```
Ignoring the parenthetical bit, it read "Some of these contribution workflows you'll need..." which doesn't feel right to me.  Alternatively we could put back the "For some...".